Makefile

```diff
@@ -351,12 +351,17 @@ test: $(JUJUGUI) $(PYTEST)
 
 .PHONY: test-js-phantom
 test-js-phantom: gui
-	./scripts/test-js.sh
+	./scripts/test-js-old.sh
 
 .PHONY: test-js-karma
 test-js-karma: gui
 	$(NODE_MODULES)/.bin/karma start karma.conf.js --single-run --browsers PhantomJS --log-level warn --reporters mocha
 
+.PHONY: test-old-js-karma
+test-old-js-karma: gui
+	./scripts/test-js.sh
+
```

Comment:
Also 'old' is a terrible name unless it will be going away very quickly, like one week.